main/isotp_ble_bridge_main.c

```diff
@@ -43,25 +43,39 @@ static const twai_filter_config_t f_config = TWAI_FILTER_CONFIG_ACCEPT_ALL();
 
 // Mutable globals for semaphores, queues, ISO-TP link
 
-static SemaphoreHandle_t isotp_task_sem;
-static SemaphoreHandle_t send_queue_start;
 static SemaphoreHandle_t done_sem;
+static SemaphoreHandle_t isotp_send_queue_sem;
+
+//static SemaphoreHandle_t isotp_task_sem; // used by isotp_processing_task
 static SemaphoreHandle_t isotp_mutex;
-static SemaphoreHandle_t isotp_wait_for_data;
+static SemaphoreHandle_t isotp_wait_for_ecu_data;
```

Comment:
@bri3d do you think this was the right thing to do? it feels pretty dumb on my end to duplicate it. i did it while i was trying to resolve some obscure pointer issue so i'm not sure if it is actually needed